preferences.yaml

```diff
@@ -0,0 +1,76 @@
+swagger: '2.0'
+info:
+  version: 0.0.1
+  title: Preferences API
+  description: >-
+    The API lets you manage attributes associated with users (user preferences).
+
+    You can get any attribute for any user (read only) according to the rights
+    you owe on the user resource. 
+
+    There rights are set up at moment you've subscribed the service exposed by
+    this API.
+  license:
+    name: MIT
+host: localhost
+basePath: /api/v1
+schemes:
+  - https
+paths:
+  '/users/{fiscal_number}/preferences':
+    x-swagger-router-controller: Preferences
+    get:
+      operationId: getUserPreferences
+      summary: Get User's Preferences
+      description: Retrieve one user's preferences (ie. email and/or preferred language).
+      responses:
+        '200':
+          description: Found.
+          schema:
+            $ref: '#/definitions/User'
+          examples: {}
+          headers:
+            ETag:
+              type: string
+              description: An opaque identifier of this resource revision.
+        '404':
+          description: User not found.
+    parameters:
+      - name: fiscal_number
+        in: path
+        type: string
+        required: true
+        description: 'The user''s fiscal number, all upper case.'
+        pattern: >-
+          [A-Z]{6}[0-9LMNPQRSTUV]{2}[ABCDEHLMPRST][0-9LMNPQRSTUV]{2}[A-Z][0-9LMNPQRSTUV]{3}[A-Z]
+        maxLength: 16
+        minLength: 1
+definitions:
+  User:
+    title: User
+    description: Describes a user
+    type: object
+    properties:
+      fiscal_number:
+        type: string
+        minLength: 1
+        maxLength: 16
+        description: 'The user''s fiscal number, all upper case.'
+        pattern: >-
+          [A-Z]{6}[0-9LMNPQRSTUV]{2}[ABCDEHLMPRST][0-9LMNPQRSTUV]{2}[A-Z][0-9LMNPQRSTUV]{3}[A-Z]
+      email:
+        type: string
+        description: |
+          The user's preferred email address.
+      preferred_languages:
+        type: array
+        description: >-
+          Indicates the User's preferred written or spoken language. Generally
```

Comment:
this is too much, not all combinations are valid / make sense, to make the life of the client simpler I would use an enum here, so they know what languages they should try to cover. I would start with:

* `it_IT`
* `en_GB`
* `es_ES`